packages/openapi-v3/src/extension-point/component.ts

```diff
@@ -0,0 +1,12 @@
+import {createBindingFromClass} from '@loopback/context';
```

Comment:
Let's use a different directory name for `extension-point`, which is generic.

Reply:
hmm, any suggestion? cuz I feel `extension-point` is generic and `spec-enhancer` is more specific

Reply:
Maybe `enhancers`